src/Symfony/Component/Workflow/TransitionBlockerList.php

```diff
@@ -37,6 +37,17 @@ public function add(TransitionBlocker $blocker): void
         $this->blockers[] = $blocker;
     }
 
+    public function has(string $code): bool
```

Comment:
Should we mark this method as internal?

Reply:
It could be usefull to any one, isn't? 
We could mark it internal, but I don't really see benefit of doing it.
It will update the code if you want.